api.go

```diff
@@ -123,11 +149,17 @@ func BuildDockerFile(p martini.Params, r render.Render, dockerInfo DockerInfo) {
 	err := GenerateDockerFile(dockerInfo, ".")
 
 	if err != nil {
-		r.JSON(500, map[string]interface{}{"success": false})
+		// this line can probably be removed but we need to figure what if anything we should return in responses
+		r.JSON(500, map[string]interface{}{"success": "false"})
+		makePutRequest(dockerInfo, "FAL")
+
 	}
 
 	logging.LogToFile("Dockerfile generated successfully...building container...")
 
 	go Build(buildLocation, dockerInfo.REPO_NAME, dockerInfo.PATH)
-	r.JSON(200, map[string]interface{}{"success": true})
+	// this line can probably be removed but we need to figure what if anything we should return in responses
+	r.JSON(200, map[string]interface{}{"success": "true"})
+	makePutRequest(dockerInfo, "BLD")
```

Comment:
I was actually going to remove that logic, and use this to update the status.  I think Success should be when when it was actually deployed.
However since we still need to write a story that functionality, I guess skipping Building and returning success makes sense for now.